sockeye/training.py

```diff
@@ -613,7 +614,7 @@ def _load_params(self, output_folder: str, checkpoint: int):
         """
         params_fname = os.path.join(output_folder, C.PARAMS_NAME % checkpoint)
         self.load_params_from_file(params_fname)  # sets self.params
-        self.module.set_params(arg_params=self.params, aux_params={})
+        self.module.set_params(arg_params=self.params, aux_params=self.aux_params)
```

Comment:
Does this work if self.aux_params is None? The documentation specifies that it expects a dictionary.

Reply:
It is correct. But `self.load_params_from_file(params_fname)` stores in `self.aux_params` an empty dictionary in case there are not auxiliary parameters.